pkg/server/skydb/query.go

```diff
@@ -385,6 +380,15 @@ func (q Query) Accept(visitor Visitor) {
 	}
 }
 
+// Query specifies the type, predicate and sorting order of Database
+// query.
+type RowACL struct {
```

Comment:
Let’s call it `AccessControlOptions`, this is because `RowACL` can be mistaken as `ACL` related to skydb.Record, 

Alternatives (in this order),

* `QueryOptions`
* `ViewAsOptions`
* `ViewOptions`
* `*ACLOptions`

You also need to change the comment.